core/tests/puppeteer-acceptance-tests/spec/voiceover-admin-tests/add-voiceover-artist-to-an-exploration.spec.ts

```diff
@@ -0,0 +1,110 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Utility functions for voiceover admin page if voiceover admin
+ * can add voiceover artist to an exploration
+ */
+
+import {UserFactory} from '../../puppeteer-testing-utilities/user-factory';
+import {VoiceoverAdmin} from '../../user-utilities/voiceover-admin-utils';
+import testConstants from '../../puppeteer-testing-utilities/test-constants';
+import {ConsoleReporter} from '../../puppeteer-testing-utilities/console-reporter';
+
+const DEFAULT_SPEC_TIMEOUT = testConstants.DEFAULT_SPEC_TIMEOUT;
+const ROLES = testConstants.Roles;
+const invalidIdErrorToastMessage =
+  'Sorry, we could not find the specified user.';
+
+ConsoleReporter.setConsoleErrorsToIgnore([
+  new RegExp(
+    'http://localhost:8181/voice_artist_management_handler/exploration/.*Failed to load resource: the server responded with a status of 400'
+  ),
+  new RegExp('Sorry, we could not find the specified user.'),
+]);
+
+describe('Voiceover Admin', function () {
+  let voiceoverAdm: VoiceoverAdmin;
+  let explorationEditor: VoiceoverAdmin;
```

Comment:
Why is the exploration editor a voiceover admin? That seems conceptually incorrect.

Reply:
Not entirely sure what you're proposing here by "new exploration creator test file", but I would expect the explorationEditor declaration here to not have a type of VoiceoverAdmin, and for that user's functions to be in a separate user utility location than for voiceover admins.

Reply:
Yeah, sure did the same